src/CONST.ts

```diff
@@ -4297,7 +4332,7 @@ const CONST = {
                     type: 'meetGuide',
                     autoCompleted: false,
                     title: 'Meet your setup specialist',
-                    description: ({adminsRoomLink}: {adminsRoomLink: string}) =>
```

Comment:
Why is this?

Reply:
@kosmydel who originally worked on this wanted to have better typescript checks for the message types so this has been removed in favor of adding a `satisfies Record<OnboardingPurposeType, OnboardingMessageType>` to the whole message CONST definition just to be clear that the values defined there are correct.